tak-web/src/main/resources/templates/pubversion/list.html

```diff
@@ -6,65 +6,213 @@
 
 <head>
     <title>Publicerade versioner - Lista</title>
-    <th:block th:insert="~{fragments/layout::css}" />
+    <th:block th:insert="~{fragments/layout::css}"/>
+    <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
```

Comment:
Ladda ner och lägg in jquery-filerna i resources så de inte behöver hämtas externt

Reply:
Fixat, koden oanvänd. Jag har tagit bort kod.